Gemfile

```diff
@@ -12,7 +12,7 @@ group :assets do
 
   # Upload assets to AWS S3
   gem 'asset_sync'
-  gem 'fog', '~>1.20', require: 'fog/aws/storage'
+  gem 'fog-aws', '~> 0.12.0', require: 'fog/aws/storage'
```

Comment:
It should relies only on aws as dependency and not all supported drivers by fog.
So bundler hasn't to install every king of fog dependencies...